lib/scrape.js

```diff
@@ -184,6 +188,33 @@ async function findOrganization({
   return null
 }
 
+async function fetchSEtag(wikidata_id) {
+  const wikiAPIurl = wdk.getEntities({
+    ids: [wikidata_id],
+    props: ['claims'],
+  })
+  const page = await fetch(wikiAPIurl)
+  let data = await page.json()
+  data = data.entities[wikidata_id]
+  //P1482 is Stack Exchange tag
```

Comment:
Can't you use a better comment style?

Reply:
Can you elaborate? I moved the comment to [here](https://github.com/coala/gci-leaders/pull/109/files#diff-e7f5b3f014790acdf91e958a37dc5f2cR20) with a little more detail.

Reply:
Redundant comment, it's a constant at the top, already has comment.